intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -2833,6 +2962,7 @@ def scan_exchange_identifier(field):
     ('Accessible-MS-RDPEUDP', 'scan_msrdpeudp', accessible_msrdpeudp),
     ('Accessible-Radmin', 'scan_radmin', accessible_radmin),
     ('Accessible-RDP', 'scan_rdp', accessible_rdp),
+    ('Accessible-RDPEUDP', 'scan_rdpeudp', accessible_rdp_eudp),
```

Comment:
What is actually the difference to the Accessible-MS-RDPEUDP feed, which we already have and is linked 4 lines above here?